src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,20 +2,26 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+    private static final char [] PASSWORD = { '1','2','3','4' };
+    private static final String INCORRECT_DATA = "Username or password are incorrect";
```

Comment:
same about `PASSWORD`
also, we don't have to use constants for exception messages